geode-wan/src/test/java/org/apache/geode/internal/cache/wan/serial/SerialGatewaySenderOperationsDUnitTest.java

```diff
@@ -266,7 +265,6 @@ public void testStopSerialGatewaySender() throws Throwable {
     vm5.invoke(() -> WANTestBase.validateQueueSizeStat("ln", 0));
   }
 
-  @Category({FlakyTest.class, WanTest.class}) // GEODE-5056
   @Test
   public void testRestartSerialGatewaySendersWhilePutting() throws Throwable {
```

Comment:
Aside from this test, are there other tests that should be written to show that this was an issue (in a non flaky way?)

Reply:
This test was introduced by me earlier to reproduce this bug